docs/cristin.ttl

```diff
@@ -467,14 +482,91 @@
     :mapstonvafield "entityDescription.contributor.role. only the first role of a person is considered." .
 
 :rollekode
-    a            rdf:Property ;
-    rdfs:comment """
+    a               :cristinfield ;
+    rdfs:comment    """
         The role of the respective Contributor for the respective publication affiliated for this role with the respective institution.
         This entry is immutable and does not change in time.
         """ ;
+    rdfs:range      xsd:string ;
+    rdfs:domain     :VARBEID_PERSON_STED_ROLLE_Class ;
+    :isignored      "false" ;
+    :mapstonvafield " maps to NVA Role" .
+
+:eierkode
```

Comment:
Same gut feeling about global definitions. "eierkode" occurs in VITENSKAPELIGARBEID_LOKAL, ARKIVFIL, and ARKIVPOST.

In the first, it distinguishes between the NVI reporting records of the respective involved institutions. In the second, it records which insitution the user who uploaded a file to an archive belongs to. In the third, I'm not sure, I think something about which institution's archive a reference to a file was found in...